external-crates/move/documentation/book/src/primitive-types/address.md

```diff
@@ -2,8 +2,10 @@
 
 `address` is a built-in type in Move that is used to represent locations (sometimes called accounts)
 in storage. An `address` value is a 256-bit (32 byte) identifier. Move uses addresses as a way to
-differentiate packages of [modules](./modules.md), where each package has its own address. Specific
-deployments of Move might also use the `address` value for [storage](./abilities.md#key) operations.
+differentiate packages of [modules](../modules.md), where each package has its own address. Specific
+deployments of Move might also use the `address` value for [storage](../abilities.md#key)
+operations. For Sui, `address` is used to represent "accounts", and also objects via strong type
+wrappers (with `sui::object::UID` and `sui::object::ID`).
```

Comment:
Another interesting place it might be nice to blockquote this out:

```suggestion
operations.

> For Sui, `address` is used to represent "accounts", and also objects via strong type
> wrappers (with `sui::object::UID` and `sui::object::ID`).
```